jsapp/js/components/processing/sidebar/sidebarDisplaySettings.module.scss

```diff
@@ -0,0 +1,46 @@
+@use '~kobo-common/src/styles/colors';
+@use 'scss/sizes';
+
+.root {
+  display: flex;
+  justify-content: right;
+  padding-top: sizes.$x18;
+  padding-bottom: sizes.$x10;
+}
+
+.description {
+  padding-bottom: sizes.$x30;
```

Comment:
Looking at the Figma file, it looks like `.description` should have something more like `40px` and `.display` should have `35px`. Let's use `sizes.$x36` that we already have for both :)